myrobot_1:
    # タスクとロボットの環境名
    task_and_robot_environment_name: 'RedMadoana-v0'
    
    # ROSワークスペースの絶対パス
    ros_ws_abspath: "/home/nabesanta/red_RL"
    
    # コントロールが実行される時間（秒）
    running_step: 0.04

    # 各コマンドごとの位置の増分
    pos_step: 0.016

    # Q学習のパラメータ
    # 学習率（α）
    alpha: 0.5
    
    # 探索率（ε）
    epsilon: 0.9
    
    # 割引率（γ）
    gamma: 0.99

    # 探索率の減衰率
    epsilon_discount: 0.999

    # エピソード数
    n_episodes: 100

    # 各エピソードにおけるステップ数
    n_steps: 5000

    # 各ステップの時間（秒）
    running_step: 0.1
